docs/modules/monitoring/proc-introducing-metrics.adoc

```diff
@@ -13,19 +13,19 @@ Add the following to the `ConfigMap` resource that defines the Kroxylicious conf
 .Minimal metrics configuration
 [source,yaml]
 ----
-adminHttp:
+management:
   endpoints:
     prometheus: {}
 ----
 
 By default, the HTTP endpoint listens on `0.0.0.0:9190`.
-You can change the hostname and port as follows:
+You can change the bind address and port as follows:
 
-.Example metrics configuration with hostname and port
+.Example metrics configuration with bind address and port
 [source,yaml]
 ----
-adminHttp:
-  host: localhost
+management:
+  bindAddress: 127.0.0.1
```

Comment:
Is it worth explaining that binding to 127.0.0.1 means its only accessible from the host? (which is perfectly fine for a container IIRC but is interesting for a bare metal install)

Reply:
I'm not sure if it the place of this documentation to document the meaning of things like the loopback address.

I'd would like to refactor the management / metric docs.  I think we can do a better job (using callouts etc) than what we have here.  Also the PodMonitor example will really belong in the Kroxylicious on Kubenetes guide.    I'll leave this for another PR.